From d37a175d2f752a856d09932ba8b9fd4e21362fdc Mon Sep 17 00:00:00 2001
From: Guillem Jover <guillem@debian.org>
Date: Fri, 20 Jun 2014 09:42:56 +0200
Subject: [PATCH] nmudiff: Send control messages inline

Do not Bcc control messages to the control@ address, we can just send
them inline now, with the Control field in the pseudo-header.
---
 scripts/nmudiff.sh | 8 ++------
 1 file changed, 2 insertions(+), 6 deletions(-)

diff --git a/scripts/nmudiff.sh b/scripts/nmudiff.sh
index dc827a1..b983ec2 100755
--- a/scripts/nmudiff.sh
+++ b/scripts/nmudiff.sh
@@ -322,20 +322,16 @@ else
 	TO_ADDRESSES_MUTT="$TO_ADDRESSES_MUTT $b@bugs.debian.org"
 	if [ "`bts select bugs:$b tag:patch`" != "$b" ]; then
 	    TAGS="$TAGS
-tags $b + patch"
+Control: tags $b + patch"
 	fi
 	if [ "$NMUDIFF_DELAY" != "0" ] && [ "`bts select bugs:$b tag:pending`" != "$b" ]; then
 	    TAGS="$TAGS
-tags $b + pending"
+Control: tags $b + pending"
 	fi
     done
     TO_ADDRESSES_SENDMAIL=$(echo "$TO_ADDRESSES_SENDMAIL" | tail -n +2)
     if [ "$TAGS" != "" ]; then
         TAGS=$(echo "$TAGS" | tail -n +2)
-        TAGS="$TAGS
-thanks"
-        BCC_ADDRESS_SENDMAIL="control@bugs.debian.org"
-        BCC_ADDRESS_MUTT="-b control@bugs.debian.org"
     fi
 fi
 
-- 
2.0.0

